### PR TITLE
Create Python-bundle-3.10.4-foss-2022a.eb

### PR DESCRIPTION
**Pull request by jhein32**
Python umbrella module which should provide users with a decent base set-up for many scientific needs.

### CODE REVIEW

File: easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+local_gccver = '11.3.0'
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+dependencies = [
+    ('Python', version),
+    ('SciPy-bundle', '2022.05'),
+    ('matplotlib', '3.5.2'),
+    ('JupyterLab', '3.5.0', '', ('GCCcore', local_gccver)),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb:19]: not needed, sub-toolchains are automatically mapped
```suggestion
    ('JupyterLab', '3.5.0'),
```

File: easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'Bundle'
+
+name = 'Python-bundle'
+version = '3.10.4'
+
+homepage = 'https://easybuild.io/'
+description = """Python distribution with a number of widely used 
+extensions incl. NumPy, SciPy, Matplotlib, JupyterLab, MPI4PY, ... """
+
+local_gccver = '11.3.0'
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb:10]: not needed, sub-toolchains are automatically mapped
```suggestion
```

Comment (jhein32) [reply easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb:10]: I hope to have some time to look at this tomorrow afternoon.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/f5b25f5f87817fd8128550eda1d72f2e for a full test report.

**Comment by jfgrimm** [comment]
see https://github.com/jhein32/easybuild-easyconfigs/pull/8

File: easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb (hunk 1 above)
Comment (jhein32) [reply easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb]: Done

File: easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb (hunk 2 above)
Comment (jhein32) [reply easybuild/easyconfigs/p/Python-bundle/Python-bundle-3.10.4-foss-2022a.eb]: Suggestion implemented.

**Comment by jhein32** [comment]
On our local cluster:

`eb Python-bundle-3.10.4-foss-2022a.eb --robot --use-existing-modules --from-pr=17626`

Builds a module that is working.

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @jhein32!

**Pull request merged**